app/helpers/manage_metadata_helper.rb

```diff
@@ -6,7 +6,7 @@ def current_manage_title
       "manage_#{resource_type}"
     elsif controller.lookup_context.prefixes.include?('manage_cmr')
       'manage_cmr'
-    elsif controller.lookup_context.prefixes.include?('manage_variables') || controller.lookup_context.prefixes.include?('variables') || controller.lookup_context.prefixes.include?('variable_drafts') || (controller.lookup_context.prefixes.include?('collection_associations') && params[:variable_id])
+    elsif controller.lookup_context.prefixes.include?('manage_variables') || controller.lookup_context.prefixes.include?('variables') || controller.lookup_context.prefixes.include?('variable_drafts') || (controller.lookup_context.prefixes.include?('collection_associations') && params[:variable_id]) || controller.lookup_context.prefixes.include?('variable_generation_processes_searches') || controller.lookup_context.prefixes.include?('variable_generation_processes')
```

Comment:
maybe consider helper method for this statement, since it is so long.

Reply:
yeah. I think I'll look at this in the next ticket.